### ISSUE TITLE
Add bold specimen

### ISSUE THREAD

**Issue by mgee** [madmalik/mononoki#7]
I know you are working on a bold version, but perhaps one could use this issue to track the progress?

**Comment by bassu** [comment]
I read the blog post about it. Is there any chance you can push the changes @madmalik ?

**Comment by madmalik** [comment]
While making the bold version I've got a little sidetracked and reworked a big portion of monoOne itself. Getting rid of visual inconsistencies, refining the hinting and yada yada. So I've had to redraw the italics and the bold version a few times. But this is the last iteration and I will publish it after it, no matter what :P 

I'm really sorry for the painfully slow pace, but type design is a painfully slow undertaking in itself, at least with a "workhorse" typeface like this. Also, this is some something like my personal "zen" project. I find it relaxing to perfecting one of the tools I'm using all day for coding, resulting in a process that is not as goal-oriented as I would normally work.
